index.js

```diff
@@ -231,7 +231,7 @@ function getFeedHtml(){
                         <div class="tweet-details">
                             <button class="tweet-detail" data-reply="${tweet.uuid}">
                                 <i class="fa-regular fa-comment-dots" data-reply="${tweet.uuid}"></i>
-                                ${tweet.replies.length}
+                                ${tweet.replies ? tweet.replies.length : '0'}
```

Comment:
This is a ternary which will print the length if its true, and zero if its false.

Reply:
Ternarys are easier to read in HTML templates because they're more terse than if statements.